### PR TITLE
Enscoresw 1481

### PR DESCRIPTION
**Pull request by premanand17**

### CODE REVIEW

File: modules/Bio/EnsEMBL/Mapper.pm [hunk 1]
```diff
@@ -358,15 +374,16 @@ sub map_coordinates {
...
-      } else {
-        $target_end =
-          $target_coord->{'end'} - ( $start - $self_coord->{'start'} );
-      }
+        # start is somewhere inside the region
+        if ( $pair->{'ori'} == 1 ) {
+          $target_start = $target_coord->{'start'} + ( $start - $self_coord->{'start'} );
+        } else {
+          $target_end = $target_coord->{'end'} - ( $start - $self_coord->{'start'} );
+        }
```
Comment (magaliruffier) [inline modules/Bio/EnsEMBL/Mapper.pm]: can we remove the extra spacing?

File: modules/Bio/EnsEMBL/Mapper.pm [hunk 2]
```diff
@@ -341,14 +348,20 @@ sub map_coordinates {
...
     my $res;
     if ( exists $pair->{'indel'} ) {
       # When next pair is an IndelPair and not a Coordinate, create the
       # new mapping Coordinate, the IndelCoordinate.
       $target_start = $target_coord->{'start'};
       $target_end   = $target_coord->{'end'};
-
+		
+      #original coordinates
+	  $ori_start = 	$self_coord->{'start'};
```
Comment (lairdm) [inline modules/Bio/EnsEMBL/Mapper.pm]: Can I be picky about spacing? We're just starting to get things looking cleaner, with all the past wonky spacing. Thanks.

File: modules/Bio/EnsEMBL/Mapper.pm (hunk 1 above)
Comment (lairdm) [inline modules/Bio/EnsEMBL/Mapper.pm]: Indeed, it all looks reasonable, and being able to truncate those trailing gaps seems useful. But I agree, would you mind just going through it again and look at your spacing, both indentation and consistent inter-variable spacing? (ie. $var1,$var2) Thanks, sorry for being picky.

File: modules/Bio/EnsEMBL/Mapper.pm [hunk 3]
```diff
@@ -385,16 +398,30 @@ sub map_coordinates {
...
+     $ori_start = ($start - $cdna_coding_start) + 1;
+     $ori_end =   $ori_start + ($target_end - $target_start);
+     
+ 
     } ## end else [ if ( exists $pair->{'indel'...})]
 
     push( @result, $res );
+    
+        
+	my $res_ori = Bio::EnsEMBL::Mapper::Coordinate->new( $self_coord->{'id'},
```
Comment (magaliruffier) [inline modules/Bio/EnsEMBL/Mapper.pm]: can we just remove the extra spacing here please?

**Comment by magaliruffier** [comment]
I think we're now good to go

**Pull request merged**